### ISSUE TITLE
Level: Much too close for comfort

### ISSUE THREAD

**Issue by tttppp** [andybalaam/rabbit-escape#513]
```
:name=Much too close for comfort
:author_name=tttppp
:author_url=https://github.com/tttppp
:num_rabbits=0
:num_to_save=1
:solution.1.code=dLcZELgO&^)xt)L tt,ZZ,!^l r[/Q^3O&J&)t
:climb=2
:brolly=1
:bridge=1
:bash=1
     ##
      #
*ttr  #
#######
:*=Ot
```

Another small world level - a simpler version of "Too close for comfort".

**Comment by colonelfazackerley** [comment]
This is very hard. My solution is the same.

I dont understand why the bridger stops.

**Comment by andybalaam** [comment]
Yeah, nice and hard, but seems fair.  The bridger stops because of rabbots' (slightly flakey) crossover-detection algorithm, which makes them stop if they think they are going to hit a rabbit.

**Comment by andybalaam** [comment]
Renamed to Much too close for comfort

**Comment by tttppp** [comment]
This seems insanely hard on my phone. I can't remember the solution, and may have to resort to decoding it!

**Comment by andybalaam** [comment]
Released in 0.11

**Comment by colonelfazackerley** [comment]
Having forgotten the solution, I independently rediscovered it...
